GUI.py

```diff
@@ -76,6 +76,9 @@ def play(self):
         while output != b'' and self.playing:
             out_stream.write(output)
             output = audio_file.readframes(self.CHUNK)
+        self.playing = False
```

Comment:
Good to know this command resets the audio. This line was used in the temporary pause button but needs to be changed.

Reply:
Glad it helped!